### DISCUSSION TITLE
JetBrains Fleet Support

### DISCUSSION THREAD

**Discussion by JackW6809** [community/community#36038] [Copilot Conversations]
Yesterday, JetBrains Fleet was released for public preview. With it seemingly heading closer and closer to a general release are there still plans to have support for Co-pilot in Fleet? As per https://github.com/community/community/discussions/8758#discussioncomment-1812655

Thanks,
Jack W.

**Comment by iamcjwilliams** [comment]
@JackW6809 Yes, Copilot will have support

**Reply by obukhov** [reply]
Is there any ETA or a page I can track status?

**Reply by xvierd** [reply]
https://blog.jetbrains.com/fleet/2023/06/fleet-1-19-ai-powered-features-and-easier-configuration-for-rust-analyzer-python-interpreters-and-npm/

**Reply by vinceglb** [reply]
Is there any update about Github Copilot support for Fleet?

**Reply by cctv1005s** [reply]
Any updates?

**Reply by Norman-else** [reply]
Any updates?

**Reply by Xiaobonor** [reply]
Any updates?

**Reply by lzp0301** [reply]
Any updates?

**Reply by Draddz1911** [reply]
Any updates?

**Reply by inannan423** [reply]
Any updates?

**Reply by Hosivi** [reply]
ANY UPDATES??

**Reply by wilkerwalace** [reply]
Hey guys, its hour to an update

**Reply by ningblue** [reply]
ANY UPDATES??

**Reply by loontail** [reply]
ANY UPDATES??

**Reply by Juan-VC** [reply]
+1

**Reply by GabrielAcostaEngler** [reply]
+1

**Reply by 1mshy** [reply]
+1

**Reply by GavinGoGaming** [reply]
+1

**Comment by erikjara** [comment]
Fleet has supported third-party plugins for weeks now. Are there any updates? Thanks in advance.

- https://www.jetbrains.com/help/fleet/managing-plugins.html
- https://github.com/JetBrains/fleet-plugin-template
- https://github.com/JetBrains/fleet-sdk-gradle-plugin

**Comment by Juan-VC** [comment]
+1

Waiting for Fleet GH Copilot support

**Comment by GavinGoGaming** [comment]
+1

When will we get this?

**Comment by nthomaske** [comment]
+1

**Comment by Yu-yu0202** [comment]
+1

**Comment by luisjose-al** [comment]
+1

**Comment by GabrielAcostaEngler** [comment]
+1

**Comment by vovapyc** [comment]
+1

**Comment by xvierd** [comment]
+1

**Comment by sihayas** [comment]
update?

**Comment by Draddy1911** [comment]
This will never happen.

**Reply by GavinGoGaming** [reply]
Its in other IJ IDEs, it'd just need a little rewrite.

**Comment by PaulekOfficial** [comment]
+1

**Comment by DrosteOfficial** [comment]
+1

**Comment by mikeechen** [comment]
+1

**Comment by Keyvan-c** [comment]
+1

**Comment by 1mshy** [comment]
+1

**Comment by cmislek** [comment]
@iamcjwilliams Any updates?
